src/main/java/io/vertx/core/http/impl/HttpServerImpl.java

```diff
@@ -637,6 +637,7 @@ private void handshake(FullHttpRequest request, Channel ch, ChannelHandlerContex
         ServerWebSocketImpl ws = new ServerWebSocketImpl(vertx, theURI.toString(), theURI.getPath(),
           theURI.getQuery(), new HeadersAdaptor(request.headers()), wsConn, shake.version() != WebSocketVersion.V00,
           connectRunnable, options.getMaxWebsocketFrameSize());
+        ws.metric = metrics.connected(wsConn.metric(), ws);
```

Comment:
My question is: What does the websocket "connected" metric represent? Does it represent only server websockets that have been connected at the server side, or client websockets connected at the client side? Or both?